addons/zeusHC/functions/fnc_createEntityDismounts.sqf

```diff
@@ -0,0 +1,81 @@
+/*
+ * Author: PabstMirror
+ * Creates a dismount squad arround a vehicle and adds parachutes for units exiting aircraft
+ */
+#include "script_component.hpp"
+
+params ["_attachedVehicle", "_newGroup", "_createUnits", "_placerOwner"];
+TRACE_4("params",_attachedVehicle, _newGroup, _createUnits,_placerOwner);
+
+
+//Exit if bad conditions and send hint back to orginal module placer:
+if (isNull _attachedVehicle) exitWith {
+    diag_log text format ["[POTATO] - Attachable not attached"];
+    ["Module needs to be attached on a vehicle"] remoteExecCall [QACEFUNC(common,displayTextStructured), _placerOwner];
```

Comment:
Maybe use bis_fnc_showCuratorFeedbackMessage instead? It doesn't matter to much, as long as the Zeus gets the feedback..